hack/run-performance-tests.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+infomsg() {
+  echo "[INFO] ${1}"
+}
+
+SETUP_ONLY="false"
+TESTS_ONLY="false"
+TEST_DATA="5"
+
+# process command line args
+while [[ $# -gt 0 ]]; do
+  key="$1"
+  case $key in
+    -so|--setup-only)
+      SETUP_ONLY="${2}"
+      if [ "${SETUP_ONLY}" != "true" -a "${SETUP_ONLY}" != "false" ]; then
+        echo "--setup-only option must be one of 'true' or 'false'"
+        exit 1
+      fi
+      shift;shift
+      ;;
+    -to|--tests-only)
+      TESTS_ONLY="${2}"
+      if [ "${TESTS_ONLY}" != "true" -a "${TESTS_ONLY}" != "false" ]; then
+        echo "--tests-only option must be one of 'true' or 'false'"
+        exit 1
+      fi
+      shift;shift
+      ;;
+    -td|--test-data)
+      TEST_DATA="${2}"
+      if [[ $2 -le 0 || $2 -ge 1000 ]]; then
+        echo "--test-data option must be is a valid number between 1 and 1000."
+        exit 1
+      fi
+      shift;shift
+      ;;
+    -h|--help)
+      cat <<HELPMSG
+Valid command line arguments:
+  -so|--setup-only <true|false>
+    If true, only setup the test environment and exit without running the tests.
+    Default: false
+  -to|--tests-only <true|false>
+    If true, only run the tests and skip the setup.
+    Default: false
+  -td|--test-data <number>
```

Comment:
should this be called "test-namespaces" because I can envision in the future we'll have additional test "data' to configure (like "test-apps" or "test-services" or "test-users" or "test-clusters" or...). Calling this "test-data" is not specific enough, especially if we expand this script in the future.
```suggestion
  -tn|--test-namespaces <number>
```